src/components/TaskclusterAuth/TaskclusterCallback.tsx

```diff
@@ -0,0 +1,129 @@
+import { useEffect, useState } from 'react';
+
+import { useSearchParams } from 'react-router-dom';
+
+// TODO remove this after integrating the auth code logic
+localStorage.setItem('requestState', 'OkCrH5isZncYqeJbRDelN');
```

Comment:
We will use sessionStorage instead of localStorage because it is a transient value. (suggested by Julien)